core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -112,27 +142,39 @@ public void reset() {
 
     @Override
     public void stop() {
-        facade.stop();
+        exoPlayer.stop();
     }
 
     @Override
     public void release() {
         getPlayerReleaseListener().onPlayerPreRelease(this);
         getStateChangedListeners().onVideoReleased();
         heart.stopBeatingHeart();
-        facade.release();
+        exoPlayer.release();
         videoContainer.hide();
     }
 
     @Override
     public void loadVideo(Uri uri, ContentType contentType) {
-        prepareForwarder.reset();
+        getPreparedListeners().reset();
         showContainer();
-        facade.prepare(uri, contentType);
+        exoPlayer.addListener(forwarder.exoPlayerEventListener());
+        exoPlayer.setVideoDebugListener(forwarder.videoRendererEventListener());
+
+        exoPlayer.setPlayWhenReady(true);
```

Comment:
what's the reason for this change? it'll break symmetry with the media player as the we're relying on 

`loadVideo -> onPrepared -> play`